### ISSUE TITLE
Docker + large library + Web import = no thumbnails?

### ISSUE THREAD

**Issue by bobobo1618** [Sonerezh/sonerezh#193]
I'm running Sonerezh in a Docker container and when I scanned through the web UI, I didn't see a single thumbnail in my library.

Things I can think of that may cause issues:
- Most (but not all) of my library is FLAC.
- I mounted the `/music` volume as read only.
- I haven't done anything special with volume permissions.
- My library is large, with ~9600 tracks.

However:
- The stats page in settings showed ~64KB of thumbnails.
- Scanning with the command line tool worked fine.

**Comment by lGuillaume124** [comment]
As the thumbnails are persistent data, did you mount the ``/thumbnails`` volume too (with rw permissions)?

**Comment by bobobo1618** [comment]
Yes, it was definitely RW. I didn't make any changes to the container between scanning through the Web UI and scanning with the command line tool, however the command line tool resulted in visible thumbnails.
